src/src/AdminBundle/Command/CreateSuperuserCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace AdminBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use AdminBundle\Entity\Admin;
+
+class CreateSuperuserCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName('admin:create-superuser')
+             ->setDescription('Creates a new Admin user')
+             ->setHelp('Create a new Administrator. Use it to log in to the admin website.')
+             ->addArgument('email', InputArgument::REQUIRED, 'E-mail of the Administrator.')
+             ->addArgument('password', InputArgument::REQUIRED, 'Password of the Administrator.');
```

Comment:
Nebolo by lepsie zadavat heslo maskovanym vstupom a nie ako parameter prikazu?

Reply:
Toto by som urobil rad, ale neviem najst ako sa to da urobit

Reply:
Tak nechajme tak zatial.